### PR TITLE
9.0 [MIG] partner_contact_department migration

### PR DESCRIPTION
**Pull request by cubells**

### CODE REVIEW

File: partner_contact_department/README.rst [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+=====
+
+To use this module, you need to:
+
+* Go to any partner's form.
+* You will find the new *Department* field below *Job Position*.
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/134/8.0
```
Comment (pedrobaeza) [inline partner_contact_department/README.rst]: Switch to 9.0

File: partner_contact_department/README.rst [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+check there if your issue has already been reported. If you spotted it first,
+help us smashing it by providing a detailed and welcomed feedback.
+
+Credits
+=======
+
+Images
+------
+
+* `Antiun Ingeniería S.L. <http://www.antiun.com>`_: Icon.
```
Comment (pedrobaeza) [inline partner_contact_department/README.rst]: This image credit is not correct. Remove it.

File: partner_contact_department/__openerp__.py [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+#                       Antonio Espinosa <antonioea@antiun.com>
+# © 2015 Antiun Ingeniería S.L. - Jairo Llopis
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+{
+    'name': 'Contact department',
+    "summary": "Assign contacts to departments",
+    'version': '9.0.1.0.0',
+    'category': 'Customer Relationship Management',
+    'author': 'Serv. Tecnol. Avanzados - Pedro M. Baeza, '
```
Comment (pedrobaeza) [inline partner_contact_department/__openerp__.py]: Remove both old company references and put `Tecnativa`

File: partner_contact_department/models/res_partner.py [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+
+class ResPartnerDepartment(models.Model):
+    _name = 'res.partner.department'
+    _order = "parent_left"
+    _parent_order = "name"
+    _parent_store = True
+    _description = "Department"
+
+    name = fields.Char(required=True, translate=True)
+    parent_id = fields.Many2one(
```
Comment (pedrobaeza) [inline partner_contact_department/models/res_partner.py:27]: Check recursion.

**Review by pedrobaeza — CHANGES_REQUESTED** [review]
Squash all the "OCA transbot" commits

**Comment by pedrobaeza** [comment]
cc @Tecnativa

**Comment by pedrobaeza** [comment]
cc @Tecnativa

**Comment by pedrobaeza** [comment]
The squash operation hasn't been correct.

File: partner_contact_department/__openerp__.py [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+#                       Antonio Espinosa <antonioea@antiun.com>
+# © 2015 Antiun Ingeniería S.L. - Jairo Llopis
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+{
+    'name': 'Contact department',
+    "summary": "Assign contacts to departments",
+    'version': '9.0.1.0.0',
+    'category': 'Customer Relationship Management',
+    'author': 'Tecnativa S.L. - Vicent Cubells, '
```
Comment (rafaelbn) [inline partner_contact_department/__openerp__.py]: Only Tecnativa

**Review by rafaelbn — CHANGES_REQUESTED** [review]

File: partner_contact_department/__openerp__.py [hunk 6]
```diff
@@ -11,7 +11,7 @@
     "summary": "Assign contacts to departments",
     'version': '9.0.1.0.0',
     'category': 'Customer Relationship Management',
-    'author': 'Tecnativa S.L. - Vicent Cubells, '
+    'author': 'Tecnativa - Vicent Cubells, '
```
Comment (pedrobaeza) [inline partner_contact_department/__openerp__.py]: Sorry, I wasn't see the second part, but it's only Tecnativa at all, also without "Vicent Cubells". Your name is put on the README in the contributors section and also on each file header.

File: partner_contact_department/__openerp__.py [hunk 7]
```diff
@@ -11,7 +11,7 @@
...
               "Odoo Community Association (OCA)",
     "license": "AGPL-3",
     'website': 'http://www.antiun.com',
```
Comment (pedrobaeza) [inline partner_contact_department/__openerp__.py]: Change also the website

**Comment by pedrobaeza** [comment]
Vicent, this one stills contains weird commits. Can you fix it?

**Comment by pedrobaeza** [comment]
Can you do a similar test than job_position?

**Comment by cubells** [comment]
Yes, I can.

**Comment by cubells** [comment]
cc @pedrobaeza 

I can't do an squash of the transbot commits. Even if I do an squash of two consecutive commits I've got an error. Weird.

File: partner_contact_department/models/res_partner.py (hunk 4 above)
Comment (yajo) [reply partner_contact_department/models/res_partner.py:27]: It seems done below.

**Review by yajo — DISMISSED** [review]
Hi, you have to use `git rebase -i origin/9.0` and follow instructions at the bottom of the file git opens to you to fixup those consecutive commits. No conflict should appear.

Also, please keep a sane history. As you might guess, having commits from 6d3c9a1ee4f53f266b2e705fb2910a2af11d46b1 to 31831450f5268ce667435b5e0bfb74fffc508f20 with the same meaningless message will not be merged until fixed. OTOH code looks alright :wink:

**Review by rafaelbn — APPROVED** [review]

**Review by rafaelbn — APPROVED** [review]
Tested in runbot

**Comment by cubells** [comment]
cc @Tecnativa

**Comment by pedrobaeza** [comment]
OK, this looks better :wink: I'll review it

**Review by pedrobaeza — APPROVED** [review]

**Pull request merged**